bootstrap/bootstrapper/fs/source_test.go

```diff
@@ -53,8 +52,8 @@ func createTempDir(t *testing.T) string {
 }
 
 func createShardDir(t *testing.T, prefix string, shard int) string {
-	shardDirPath := path.Join(prefix, strconv.Itoa(shard))
-	err := os.Mkdir(shardDirPath, os.ModeDir|os.FileMode(0755))
+	shardDirPath := fs.ShardDirPath(prefix, uint32(shard))
```

Comment:
Probably will have some conflicts here since I got rid of `createShardDir` in my diff :\ fwiw I moved all the filepath related logic in this test file to three simple functions so hopefully should be straightforward to fix them up.